src/ventus/wm/window.js

```diff
@@ -295,9 +297,11 @@ function(Emitter, Promise, View, WindowTemplate) {
 		set maximized(value) {
 			if(value) {
 				this._restoreMaximized = this.stamp();
+				this.el.addClass('maximized');
 				this.signals.emit('maximize', this, this._restoreMaximized);
 			}
 			else {
+				this.el.removeClass('maximized');
```

Comment:
This is good, but I realized there are cases where the classes of the element do not reflect the state of the window. For example, if you maximize it and then minimize, this code is not executed and the class ```maximized``` will not be removed.

We need to completely change the way classes and state is synced (an issue not trivial to solve with this "vanilla js" library).

The bug was already there so I don't think it is a blocker for this PR. Thanks for doing this!